Core/SyncBookmarksAdapter.swift

```diff
@@ -35,21 +35,26 @@ public final class SyncBookmarksAdapter {
     public private(set) var provider: BookmarksProvider?
     public let databaseCleaner: BookmarkDatabaseCleaner
     public let syncDidCompletePublisher: AnyPublisher<Void, Never>
+    public let widgetRefreshCancellable: AnyCancellable
+    public static let syncBookmarksPausedStateChanged = Notification.Name("com.duckduckgo.app.SyncPausedStateChanged")
+    public static let shouldShowBookmarkPauseError = Notification.Name("com.duckduckgo.app.ShowSyncBookmarksPausedError")
```

Comment:
As a notification, the name should express what happened, not how it should be handled. So I'd suggest calling it "bookmarksSyncLimitReached". Same applies to Credentials.